test/test.js

```diff
@@ -348,15 +368,9 @@ describe( 'rollup', function () {
 				( config.skip ? it.skip : config.solo ? it.only : it )( dir, function ( done ) {
 					process.chdir( path.resolve( CLI, dir ) );
 
-					if (os.platform() === 'win32') {
-						config.command = "node " + path.resolve( __dirname, '../bin' ) + path.sep + config.command;
-					}
+					const command = 'node ' + path.resolve( __dirname, '../bin' ) + path.sep + config.command;
```

Comment:
Huh – yeah, I suppose you probably could. Just seems a bit weird to pass non-path things (i.e. `--flags` and other options) to `path.resolve` – am worried it might end up mangling the options somehow